src/specBuilder/bullet/markUtils.test.ts

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Can you rename this file `bulletMarkUtils.ts`?
We like to have unique names for every file so that when you use CMD + P to find files, it's very clear what they are for.